src/components/Input/Input.jsx

```diff
@@ -5,7 +5,7 @@ import styles from "./Input.module.scss";
 export const Input = React.forwardRef(
   ({ label, errorMessage, ...rest }, ref) => (
     <label className={styles.fieldWrapper}>
-      <div>{label}</div>
+      {!rest.hidden && <div>{label}</div>}
```

Comment:
Maybe it would be better to render label only when it's provided? Instead of introducing ninja-`hidden` prop?
```suggestion
      {!label && <div>{label}</div>}
```

Reply:
It would make sense if I'd write a suggestion correctly 🤦 
The correct form should go without `!` (`{label && <div>{label}</div>}`) to display the label when it's provided.